lib/apiservers/engine/backends/container_test.go

```diff
@@ -615,7 +616,7 @@ func TestPortInformation(t *testing.T) {
 	mockContainerInfo := &plmodels.ContainerInfo{}
 	mockContainerConfig := &plmodels.ContainerConfig{}
 	containerID := "foo"
-	mockContainerConfig.ContainerID = &containerID
+	mockContainerConfig.ContainerID = containerID
```

Comment:
@hmahmood the good part of this change is that it will expose problematic parts of our API and data checks earlier. What really worries me is that we do not really care about data validation relaying purely on a sane usage. At the moment it is fine, but AFAIK PortLayer API will be exposed in the future, so absence of data validation will bite us.